### ISSUE TITLE
error starting plugin at sonar 6.1:alpine

### ISSUE THREAD

**Issue by allustin** [javamelody/sonar-javamelody#4]
plugin does not start

stacktrace is

```
java.lang.UnsupportedOperationException: Sessions are disabled so that web server is stateless
sonarqube_1  |  at org.sonar.server.platform.web.RootFilter$ServletRequestWrapper.notSupported(RootFilter.java:159) ~[sonar-server-6.1.jar:na]
sonarqube_1  |  at org.sonar.server.platform.web.RootFilter$ServletRequestWrapper.getSession(RootFilter.java:150) ~[sonar-server-6.1.jar:na]
sonarqube_1  |  at net.bull.javamelody.MonitoringFilter.putUserInfoInSession(MonitoringFilter.java:300) ~[na:na]
sonarqube_1  |  at net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:232) ~[na:na]
sonarqube_1  |  at net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:178) ~[na:na]
```

step to reproduce is simple
- start the official sonar docker image 

```
sonarqube:                
  image: sonarqube:alpine 
  ports:                  
    - "9000:9000"           
    - "9092:9092"         
```
- mount the plugin dir from docker containers

```
 volumes:
      - /srv/sonar/extensions:/opt/sonarqube/extensions
```
- wget the release to extension dir https://github.com/javamelody/sonar-javamelody/releases/tag/1.62.0
- start the docker and you will see this page 

![image](https://cloud.githubusercontent.com/assets/55323/19370248/592f23fe-91b3-11e6-9a2e-fba3692735c1.png)

**Comment by evernat** [comment]
I have created [Sonar PR 1305](https://github.com/SonarSource/sonarqube/pull/1305) to fix this issue.

**Comment by evernat** [comment]
Sonar PR 1305 is merged a long time ago and released in Sonar v6.2.
